bower.json

```diff
@@ -1,6 +1,32 @@
 {
-	"name": "Ripple.js",
-	"author": "Jacob Kelley",
-	"version": "1.2.0",
-	"main": "dist/ripple.min.js"
-}
\ No newline at end of file
+  "name": "Ripple.js",
+  "version": "1.2.0",
+  "homepage": "http://jakiestfu.github.io/Ripple.js/demo",
+  "authors": [
+    "Jacob Kelley <jakie8@gmail.com>"
+  ],
+  "description": "Adds Material style ripple to anything",
+  "main": [
+    "src/ripple.js",
+    "src/ripple.css"
+  ],
```

Comment:
So you may do non-minified and minified in dist folder. And both with your header.